code/core/src/common/js-package-manager/JsPackageManager.ts

```diff
@@ -380,6 +380,10 @@ export abstract class JsPackageManager {
 
         const isLatestStableRelease = currentVersion === latestInRange;
 
+        if (latestInRange?.includes('-local.')) {
+          return `${packageName}@local`;
+        }
```

Comment:
The `latestInRange` is a lookup of what's on the `latest` tag on the current registry.
It including `-local` would only ever be the case when running verdaccio, right?

So I'm guessing `latest` and `local` tags are actually equal on verdaccio.
So what is the point of converting it to `local` tag?

Reply:
We have tests for this code IIRC, we should ensure added behavior has tests in place.